src/theme/components/button.ts

```diff
@@ -3,19 +3,82 @@ import { defineStyleConfig } from "@chakra-ui/react";
 export const buttonTheme = defineStyleConfig({
   baseStyle: {
     backgroundColor: "primary.600",
+    borderRadius: "base",
+    // token?
     color: "white",
     px: 6,
     py: 3,
   },
   sizes: {
-    sm: {},
-    md: {},
-    lg: {},
+    sm: {
+      height: 11,
+    },
+    md: {
+      height: 12,
+    },
+    lg: {
+      height: 14,
+    },
   },
   variants: {
-    primary: {},
-    secondary: {},
-    tertiary: {},
+    primary: {
+      // token?
+      boxShadow: "0 1.5 1.5 0 rgba(35, 78, 82, 0.08)",
+      _disabled: {
+        backgroundColor: "primary.500",
+        color: "white",
+        // token?
+        opacity: 0.64,
+      },
+      _hover: {
+        backgroundColor: "brand.50",
+        // token?
+        border: "1px solid",
+        borderColor: "brand.800",
```

Comment:
You're correct that Chakra doesn't recognize color tokens when you pass them into `border` directly. I think that's actually intentional? As in - they want you to pass the thickness and type (dashed, solid, etc) as `border` and the color as `borderColor`.

For now, they way you're doing it here is great - set border with raw values and the color with a color token. We can revisit `border` tokens later and refactor this.